src/cmd/manage_nsfs.js

```diff
@@ -0,0 +1,411 @@
+            tag: "",
+            versioning: "DISABLED",
+            path: bucket_path,
+            should_create_underlying_storage: true,
+        };
+    }
+    if (action === 'update') {
+        data = _.omitBy(data, _.isUndefined);
+        data = update_bucket(config_root, data);
+    }
+    return data;
+}
+
+async function update_bucket(config_root, target) {
+    const account_path = config_root + '/buckets';
+    let source;
+    try {
+        const source_raw = await fs.promises.readFile(path.join(account_path, target.name + '.json'));
+        source = JSON.parse(source_raw.toString());
+        //target.nsfs_account_config.new_buckets_path = target.nsfs_account_config.new_buckets_path || source.nsfs_account_config.new_buckets_path;
+    } catch (err) {
+        console.error('NSFS Manage command: Could not find bucket ' + target.name + ' to update');
+        print_account_usage();
+    }
+    const data = _.merge({}, source, target);
+    return data;
+}
+
+async function manage_bucket_operations(action, data, config_root) {
+    const bucket_config_path = config_root + '/buckets';
+    if (action === 'add') {
+        const is_valid = await validate_bucket_add_args(data);
+        if (!is_valid) {
+            print_bucket_usage();
+            return;
+        }
+        create_resourse(data.name, bucket_config_path, data);
+    } else if (action === 'status') {
+        const is_valid = await validate_minimum_bucket_args(data);
+        if (!is_valid) {
+            print_account_usage();
+            return;
+        }
+        const bucket_dir_stat = await fs_utils.file_exists(path.join(bucket_config_path, data.name + '.json'));
+        if (bucket_dir_stat) {
+            console.log("Bucket exists with access key : " + data.name);
+        } else {
+            console.log("Bucket do not exists with access key : " + data.name);
+        }
+    } else if (action === 'update') {
+        const is_valid = await validate_bucket_add_args(data);
+        if (!is_valid) {
+            print_bucket_usage();
+        }
+        create_resourse(data.name, bucket_config_path, data);
+    } else if (action === 'delete') {
+        const is_valid = await validate_minimum_bucket_args(data);
+        if (!is_valid) {
+            print_bucket_usage();
+        }
+        delete_resourse(data.name, bucket_config_path);
+    } else if (action === 'list') {
+        const buckets = await list_resources(bucket_config_path);
+        console.log(buckets);
+    } else {
+        print_bucket_usage();
+    }
+}
+
+async function account_management(argv) {
+    const action = argv._[1] || '';
+    const config_root = argv.config_root ? String(argv.config_root) : '';
+    const data = await fetch_account_data(argv);
+    manage_account_operations(action, data, config_root);
+}
+
+async function fetch_account_data(argv) {
+    let data;
+    const action = argv._[1] || '';
+    const from_file = argv.from_file ? String(argv.from_file) : '';
```

Comment:
it looks like from_file and config_root assignment can be done once in main, WDYT? 
const config_root = argv.config_root ? String(argv.config_root) : ''; is done many times and I don't think it's necassary